tests/Php81/fixtures/curl-echo-server.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+$output = $_FILES;
+foreach ($_FILES as $name => $file) {
+    unset($file['tmp_name']);
+    $output[$name] = $file;
+}
+print_r($output);
```

Comment:
I'd rather use `json_encode` than `print_r`. I'm not sure `print_r` is guaranteed to keep the exact same output formatting forever.

Reply:
Thank you; You are right, `json_encode` is more resilient. Updating now.